ddtrace/contrib/django/migration.md

```diff
@@ -0,0 +1,91 @@
+# Migrating to the new Django integration
+
+We have shifted from using a Django app style of configuration to using the
+method consistent with our other integrations.
+
+## Steps
+
+1. Remove `'ddtrace.config.django'` from `INSTALLED_APPS` in `settings.py`.
+
+2. Replace `DATADOG_TRACE` configuration in `settings.py` according to the table below.
+
+See below for the mapping from old configuration settings to new ones.
+
+| DATADOG_TRACE setting | Replacement         |
+|-----------------------|---------------------|
+| `AGENT_HOSTNAME` | `DD_AGENT_HOST` environment variable or `tracer.configure(hostname=)` |
+| `AGENT_PORT`     | `DD_TRACE_AGENT_PORT` environment variable or `tracer.configure(port=)` |
+| `AUTO_INSTRUMENT`| N/A Instrumentation is automatic |
+| `INSTRUMENT_CACHE` | N/A Instrumentation is automatic |
+| `INSTRUMENT_DATABASE` | N/A Instrumentation is automatic |
```

Comment:
Please make this opt-out rather than automatic :)

Reply:
Is there a reason why you would disable it?

do you currently have this disabled?

Reply:
For one or two of our workers that do a lot of work, having this on caused memory leaks. So we have it on for most things but not those ones.

Reply:
👍